### DISCUSSION TITLE
Scan for correspondents / Tags / Document types

### DISCUSSION THREAD

**Discussion by Pheggas** [jonaswinkler/paperless-ng#1461] [Feature Requests]
Hello. Recently, i finally got paperless-ng working. From this point, i started to throw there all the documents that i have mindlessly. After then, i noticed there's correspondents etc. button where i can add correspondent. The problem here is i can't somehow re-scan the library so i can use this feature. It can be applied for only new documents that will be scanned in the future.

I'd like to have re-scan button (or Scan for correspondents / Tags / Document types) button

**Comment by philosowaffle** [comment]
You can do this using the `document_retagger` from the command line: https://paperless-ng.readthedocs.io/en/latest/administration.html#document-retagger

**Reply by torwag** [reply]
You could also mark documents in the document overview and give them all together the same tag or document type. This goes quite quickly, albeit it is sometimes difficult to see the type of document in the small preview.
